lesson 0/README.md

```diff
@@ -11,7 +11,8 @@ This is the recommended OS for this course. If you don't already have Ubuntu on
 If you do this before class we'll have a lot more time to help you out and get started! Please download [Android Studio 2.1](https://developer.android.com/studio/install.html).
 
 *Note*: For those of you who already have Android Studio, it's important that you have the latest version. This makes it easier for us to help you with any problems you might have.
-*Note 2*: If you choose to use Android Studio on Windows, then you'll need to download github desktop [link here].
+
+*Note 2*: If you choose to use Android Studio on Windows or OSX and don't know how to get git integration with your terminal, then you'll can download [Github Desktop](https://desktop.github.com/) instead.
```

Comment:
I think that we should really encourage using git from the command line. It's a software development class, and as far as our learning goals go using the command line is better aligned.